cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,9 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(LINUX_DESKTOP_FILE_EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms %f")
+SET(MENU_FILE_ICON_XPM "/usr/share/pixmaps/lmms.xpm")
+SET(LINUX_RELATIVE_DESKTOP_FILE_ICON "lmms")
+CONFIGURE_FILE(lmms.desktop.in ${CMAKE_PROJECT_NAME}.desktop)
+CONFIGURE_FILE(menu.in ${CMAKE_PROJECT_NAME})
```

Comment:
I don't think we should place a file anywhere that is simply called `LMMS` with no file extension.  Can you link me to a document that says the menu system requires this?